Sources/StreamVideo/WebRTC/VideoCapturingUtils.swift

```diff
@@ -0,0 +1,107 @@
+//
+// Copyright © 2023 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import WebRTC
+
+class VideoCapturingUtils {
```

Comment:
would it make more sense to have it as enum as this isn't supposed to be initialized?

Reply:
sure, updated